### ISSUE TITLE
feat: rpc server supports QUIC connections

### ISSUE THREAD

**Issue by kckeiks** [fleek-network/ursa#304]
# Description

Blocked by hyper not supporting QUIC/HTTP3 yet.

**Comment by ozwaldorf** [comment]
Maybe something nice to track: https://github.com/hyperium/h3

It's purpose is to be used internally by hyper once it's stable

**Comment by kckeiks** [comment]
Yup and it's blocking this too https://github.com/seanmonstar/reqwest/pull/1599.

**Comment by kckeiks** [comment]
> Yup and it's blocking this too [seanmonstar/reqwest#1599](https://github.com/seanmonstar/reqwest/pull/1599).

Another one to track https://github.com/hyperium/hyper/issues/1818.

**Comment by ozwaldorf** [comment]
just for visibility we also have this https://github.com/aws/s2n-quic
